apps/yapms/src/lib/components/sidebar/sections/sources/Sources.svelte

```diff
@@ -1,18 +1,20 @@
 <script lang="ts">
 	import { SideBarStore } from '$lib/stores/SideBar';
 
-	$: sources = $SideBarStore.sources.map((source) => {
-		return new URL(source);
-	});
+	let sources = $derived(
+		$SideBarStore.sources.map((source) => {
+			return new URL(source);
+		})
+	);
 </script>
 
 <div class="divider">Sources</div>
-<ul class="flex flex-col gap-4 p-4">
+<ul class="flex flex-col gap-4 m-4">
```

Comment:
This should have "items-center" to keep links centered to be consistent with the rest of the sidebar UI sections.